### PR TITLE
Converted BrineCo2Pvt.hpp to UTF-8

### PR DESCRIPTION
**Pull request by blattms**

### CODE REVIEW

File: opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp [hunk 1]
```diff
@@ -352,7 +352,7 @@ class BrineCo2Pvt
         Scalar M_CO2 = CO2::molarMass();
         Scalar M_H2O = H2O::molarMass();
 
-        const LhsEval& tempC = temperature - 273.15;        /* tempC : temperature in �C */
+        const LhsEval& tempC = temperature - 273.15;        /* tempC : temperature in °C */
```
Comment (bska) [inline opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp:355]: I doesn't show in the GitHub review interface, but I **assume** you've only changed the encoding of the degrees Centigrade symbol here, right?

Comment (bska) [inline opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp:355]: I doesn't show in the GitHub review interface, but I **assume** you've only changed the encoding of the degrees Centigrade symbol here, right?

**Comment by totto82** [comment]
Thanks have notices the issue in the master and trust you that this is a fix. Will merge....

**Comment by blattms** [comment]
Thanks. iconv should only have changed what was needed (read single characters). Hence I assume you are right.

**Pull request merged**